source/DonutModel.h

```diff
@@ -38,6 +38,10 @@ class DonutModel {
    protected:
 	/** INITIAL position of the donut in world space */
 	cugl::Vec2 initial;
+	/** Position of the ship in world space */
+	cugl::Vec2 position;
```

Comment:
What is your intended use for the position variable? Right now the donut's position is entirely determined by its angle since it can only move around a circle.

Reply:
That was an oversight on my part, thanks for catching it

Reply:
Add a commit to the pr real quick and I'll approve